### ISSUE TITLE
[topic] tox.ini

### ISSUE THREAD

**Issue by isabelizimm** [isabelizimm/cookiecutter-custompy#4]
tldr;
`setup.cfg`/`setup.py` are for `setuptools`, and `tox.ini` is for `tox`, but uses `setuptools` to build the distribution.

[Tox](https://tox.wiki/en/latest/) the Python project is made to "automate and standardize testing in Python." It creates virtual environments, and installs the configured dependencies for those environments, for the purpose of testing a Python package. A `tox.ini`, `pyproject.toml`, and `setup.cfg` are all [ini files](https://en.wikipedia.org/wiki/INI_file).

Why tox over GitHub Actions? Do we need both?

[1] https://www.integralist.co.uk/posts/toxini/
[2] https://stackoverflow.com/questions/67098213/is-there-a-reason-to-have-both-tox-ini-and-setup-cfg-in-a-python-project
